assets/sass/patterns/organisms/listing.scss

```diff
@@ -34,6 +34,21 @@ $totalHighlightsSpacing: 6;
   .content-header--read-more {
     border: none;
   }
+
+  // When wide, hide to avoid duplication: then, the contents is also displayed in the right hand article column
+  .listing-list__item--related:first-child {
```

Comment:
I'd say either lose the `:first-child` and allow for any item being related (unlikely), or be explicit and use something like `.listing-list--has-related > :first-child`.